tests/test_create_inconsistent_dimensionality.py

```diff
@@ -38,16 +40,18 @@
 def test_create_from_geojson(geojson):
     with pytest.raises(ValueError) as exc:
         wkt = shape(geojson).wkt
-    assert exc.match("Inconsistent coordinate dimensionality")
+    assert exc.match("Inconsistent coordinate dimensionality|Input operand 0 does not have enough dimensions")
```

Comment:
We get different error message in several places (we should see to what extent we want to preserve them, sometimes they are more informative though)